doc/cascadia/SettingsSchema.md

```diff
@@ -3,43 +3,43 @@
-| `startingDirectory` | _Required_ | String | The directory the shell starts in when it is loaded. |
-| `useAcrylic` | _Required_ | Boolean | When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. |
-| `colorTable` | Optional | Array[String] | Array of colors used in the profile if `colorscheme` is not set. Colors use hex color format: `"#rrggbb"`. Ordering is as follows: `[black, red, green, yellow, blue, magenta, cyan, white, bright black, bright red, bright green, bright yellow, bright blue, bright magenta, bright cyan, bright white]` |
-| `cursorHeight` | Optional | Integer | Sets the percentage height of the cursor starting from the bottom. Only works when `cursorShape` is set to `"vintage"`. Accepts values from 25-100. |
-| `foreground` | Optional | String | Sets the foreground color of the profile. Overrides `foreground` set in color scheme if `colorscheme` is set. Uses hex color format: `"#rrggbb"`. |
-| `icon` | Optional | String | Image file location of the icon used in the profile. Displays within the tab and the dropdown menu. |
-| `scrollbarState` | Optional | String | Defines the visibility of the scrollbar. Possible values: `"visible"`, `"hidden"` |
-| `tabTitle` | Optional | String | Overrides default title of the tab. |
+| Property | Necessity | Type | Default | Description |
+| -------- | --------- | ---- | ------- | ----------- |
+| `acrylicOpacity` | _Required_ | Number | `0.5` | When `useAcrylic` is set to `true`, it sets the transparency of the window for the profile. Accepts floating point values from 0-1. |
+| `closeOnExit` | _Required_ | Boolean | `true` | When set to `true`, the selected tab closes when `exit` is typed. When set to `false`, the tab will remain open when `exit` is typed. |
+| `colorScheme` | _Required_ | String | `Campbell` | Name of the terminal color scheme to use. Color schemes are defined under `schemes`. |
+| `commandline` | _Required_ | String | `powershell.exe` | Executable used in the profile. |
+| `cursorColor` | _Required_ | String | `#FFFFFF` | Sets the cursor color for the profile. Uses hex color format: `"#rrggbb"`. |
+| `cursorShape` | _Required_ | String | `bar` | Sets the cursor shape for the profile. Possible values: `"vintage"` ( &#x2583; ), `"bar"` ( &#x2503; ), `"underscore"` ( &#x2581; ), `"filledBox"` ( &#x2588; ), `"emptyBox"` ( &#x25AF; ) |
+| `fontFace` | _Required_ | String | `Consolas` | Name of the font face used in the profile. |
+| `fontSize` | _Required_ | Integer | `10` | Sets the font size. |
+| `guid` | _Required_ | String | | Unique identifier of the profile. Written in registry format: `"{00000000-0000-0000-0000-000000000000}"`. |
+| `historySize` | _Required_ | Integer | `9001` | The number of lines above the ones displayed in the window you can scroll back to. |
+| `name` | _Required_ | String | `PowerShell Core` | Name of the profile. Displays in the dropdown menu. |

```

Comment:
There is no default here really

Reply:
UPDATING SISTEM FOR CC CHRT WALTER 
ELRUSSORD*

Reply:
Should I keep PowerShell guid in globals for defaultProfile?